server/middleware/grpc-client/transform/cli-transform.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021-2022 Uber Technologies Inc.
+//
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+const Long = require('long');
+const losslessJSON = require('lossless-json');
+
+const cliTransform = item => {
```

Comment:
what do we use this for? Is this something we should cover too?

Reply:
these are specific transforms for export json to CLI. These have been used in tchannel implementation and hence why it was copied to grpc. Basically are some extra transforms on the returned json to make sure they match the CLI implementation of exporting history. I will add a unit test for this one.

Reply:
unit tests added.

Reply:
sweet